packages/rest/test/acceptance/bootstrapping/rest.acceptance.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect, supertest} from '@loopback/testlab';
+import {Constructor, Provider, inject} from '@loopback/context';
+import {
+  DefaultSequence,
+  RestBindings,
+  RestServer,
+  RestComponent,
+} from '../../..';
+import {Application, ControllerClass} from '@loopback/core';
+
+describe('Bootstrapping with RestComponent', () => {
+  context('with a user-defined sequence', () => {
+    let app: Application;
+    let server: RestServer;
+    before(givenAppWithUserDefinedSequence);
+
+    it('binds the `sequence` key to the user-defined sequence', async () => {
+      const binding = await server.get(RestBindings.SEQUENCE);
```

Comment:
This does not work on my computer. `UserDefinedSequence` depends on a bunch of elements, starting with `http.request.context` and ending with sequence actions.

While the sequence actions should be bound at app/server level (they are not yet!), the current request context will be always request-local.

Here is a custom sequence that works for me:

```ts
      class UserDefinedSequence implements SequenceHandler {
        async handle(
          request: ParsedRequest,
          response: ServerResponse,
        ): Promise<void> {
          response.statusCode = 500;
          response.end('Not Implemented');
        }
      }
```

Reply:
Uh oh, ignore my comment. Looks like this test is instantiating `DefaultSequence`, which is caused by the fact that I am investigating the problem why `#rest` sub-key does not work.